rootfs/etc/nginx/lua/balancer.lua

```diff
@@ -36,6 +36,8 @@ local IMPLEMENTATIONS = {
 
 local _M = {}
 local balancers = {}
+local external_backends = {}
+local last_timestamp = 0
```

Comment:
I think this should be more specific and indicate that it is specifically about backend sync - what about `backends_last_synced_at`?